projects/akita-ng-fire/src/lib/collection/collection.service.ts

```diff
@@ -105,20 +107,20 @@ export class CollectionService<
     key: string | Query,
     cb: () => Observable<any>
   ): Observable<any> {
-    if (!this.useMemorization) return cb();
+    if (!this.useMemorization) { return cb(); }
```

Comment:
no need to wrap with braces

Reply:
I've added those because of `TSLint` rule. Anyway, updated the rule and removed the braces.